assignment.js

```diff
@@ -65,7 +67,8 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
-
+let numbers = [57, 13, 4, 28]
+console.log(Math.max(...numbers))
```

Comment:
@kyleligman nice use of the es6 rest parameter to pass an array as an argument to `Math.max()`